ts/screens/wallet/AddCardScreen.tsx

```diff
@@ -63,6 +64,40 @@ export default class AddCardScreen extends React.Component<Props, State> {
     };
   }
 
+  private submit = () => {
+
+    console.warn("let's go!");
+    const { pan, expirationDate, securityCode, holder } = this.state;
```

Comment:
I would extract the validation logic here and use it to:

* provide visual feedback to the user as she completes the form
* enable/disable the submit button (you don't want to validate on submit, you want the user to be able to submit only on valid data)

Reply:
Yep that makes absolute sense. I have just now finished working on the full process integration. Will now work on this aspect